Chapter_2_Getting_Started.md

```diff
@@ -0,0 +1,1206 @@
+# 2 Getting Started with Robot Framework
+
+This chapter introduces participants to the foundational concepts of Robot Framework.
+It covers the basics of how automation specifications are structured, how suites are organized, and the execution process.
+Participants will learn how Robot Framework is run and explore the generated reports and logs that document test results.
+
+The chapter also provides an overview of suite structures,
+the role of libraries and resource files,
+and how to import them.
+Additionally, it delves into the core syntax of Robot Framework,
+focusing on how keywords are defined and used, the types of keyword arguments,
+and how keyword documentation is interpreted to ensure clarity and maintainability.
+
+
+
+
+## 2.1 Suite File & Tree Structure
+
+> [!IMPORTANT]
+> LO-2.1 Understand which files and directories are considered suites and how they are structured in a suite tree. (K2)
+
+When executing Robot Framework, it either parses directory trees or files, depending on which paths are given.
+
+The given path to Robot Framework where it starts parsing is considered the **Root Suite**.
+
+If the path to a single file is given as **Root Suite** directly to Robot Framework, only this file is parsed.
+
+If a directory path is given, starting at this location, Robot Framework will parse all `*.robot` files and directories within this path.
+Robot Framework analyzes all containing files and determines if they contain test cases or tasks. If they do, they are considered **Suite Files** or **Low-Level Suites**.
+All directories that either directly or indirectly contain a Suite File are considered **Suites Directories** or **Higher-Level Suites**.
+
+The ordering of suites during execution is, by default, defined by their name and hierarchy.
+All files and directories, which are suites in one directory, are considered on the same level and are executed in case-insensitive alphabetical order.
+
+
+It is possible to define the order without influencing the name of the suite by adding a prefix followed by two underscores `__` to the name of the directory or file. This prefix is NOT considered part of the name.
+So `01__First_Suite.robot` sets the suite name to `First Suite`, while `2_Second_Suite.robot` sets the suite name to `2 Second Suite`.
+
+One or more underscores in file or directory names are replaced by space characters as suite names.
+
+Legend:
+```plaintext
+▷ Directory (No Suite)
+▶︎ Suite Directory
+◻︎ File (No Suite)
+◼︎ Suite File
+```
+
+Example:
+```plaintext
+ ----- Tree Structure / Order --------- | ---- Suite Name ---------
+  ▶︎ Root_Suite                          | Root Suite
+    ◼︎ A_Suite.robot                     | A Suite
+    ▶︎ Earlier_Suite_Directory           | Earlier Suite Directory
+      ◼︎ B_Suite.robot                   | B Suite
+      ◼︎ C_Suite.robot                   | C Suite
+    ▷ Keywords (No Suite)               |
+      ◻︎ technical_keywords.resource     |
```

Comment:
`.resource` extension has not been introduced before this "figure"

Reply:
@datakurre Imho that doesn't really matter, because any extension except .robot is not parsed by default and therefore not considered to be a suite file.